doc/Web_Test_Suite_Packaging_Guide.md

```diff
@@ -1,604 +1,208 @@
+        $ sudo npm install -g crosswalk-app-tools 
 
-Set packaging environment for Android:
-
-- Install ant nodejs:
-
-    $ sudo apt-get install ant nodejs
-
-- Download and install Android SDK(>=android-19) for your platform from [http://developer.android.com/sdk/index.html](http://developer.android.com/sdk/index.html).
+- Download and install Android SDK(>=android-19) for your platform from <http://developer.android.com/sdk/index.html>.
 - Deploy Android SDK`s tools and platform-tools to PATH environment
 
-    $ export PATH=${PATH}:/path/to/adt-bundle-<version\>/sdk/platform-tools:/path/to/adb-bundle-/sdk/platform-tools
-
-There are two tools to generate APK package, make\_apk.py and cordova.
-
-make\_apk.py:
-
-- Get proper package of <version\>/crosswalk-<version\>.zip from [https://download.01.org/crosswalk/releases/crosswalk/android/canary/](https://download.01.org/crosswalk/releases/crosswalk/android/canary/).
-
-    $ cd /path/to/crosswalk-test-suite/tools/
-
-    $ unzip /path/to/crosswalk-<version\>.zip
-
-    $ mv crosswalk-<version\>\* crosswalk/
-
-    make\_apk.py is located in the crosswalk folder.
-
-cordova:
-
-- Get proper package of <version\>/<arch\>/crosswalk-cordova-<version\>-<arch\>.zip from [https://download.01.org/crosswalk/releases/crosswalk/android/canary/](https://download.01.org/crosswalk/releases/crosswalk/android/canary/).
-
-    $ cd /path/to/crosswalk-test-suite/tools/
-
-    $ unzip /path/to/crosswalk-cordova-<version\>-<arch\>.zip
-
-    $ mv crosswalk-cordova-<version\>-<arch\> cordova
-
-## 3. Pack Web Test Suite Packages
-
-There is a pack.sh script in each test suite. Currently it supports 3 types of test suite packages, APK, XPK and WGT, in .zip, for 3 platforms, Android, Tizen Mobile and IVI.
-
-### 3.1  Pack Web Test Suite Packages for Android
-
-Pack APK packages use make\_apk.py:
-
-    $ ./pack.sh –t apk –m embedded –a arm|x86
-
-   Pack embedded mode APK package using make\_apk.py. use –a arm or –a x86 to choose the right architecture.
-
-    $ ./pack.sh –t apk –m shared
-
-   Pack shared mode APK package using make\_apk.py.
-
+        $ export PATH=$PATH:/path/to/adt-bundle-<version>/sdk/platform-tools:/path/to/adb-bundle-/sdk/platform-tools
+
+- Install ant maven (for android)
+
+        $ sudo apt-get install ant maven
+
+- Set a environment variable named CROSSWALK_APP_TOOLS_CACHE_DIR:
+
+        $ export CROSSWALK_APP_TOOLS_CACHE_DIR=/custom/your/environment/path/
+
+- Get proper package of `<version>/crosswalk-<version>.zip` from <https://download.01.org/crosswalk/releases/crosswalk/android/canary/>. 
+    
+        $ mv crosswalk-<version>.zip $CROSSWALK_APP_TOOLS_CACHE_DIR
+
+Besides app-tools, cordova tool is also support to build Web Apps for Android. 
+
+- if build with cordova, Get proper package of `<version>/<arch>/crosswalk-cordova-<version>-<arch>.zip` from <https://download.01.org/crosswalk/releases/crosswalk/android/canary/>.
```

Comment:
@qiuzhong feel free to update this in another pull request.

I would like to merge this one so that you can make further improvement, thanks.